### PR TITLE
fix: correct course 3 of algebra

### PR DESCRIPTION
**Pull request by GabinDDL**
# fix: correct course 3

## Description

Corrects typo errors in Week 3 lessons on polynomial functions and irreducibles

## Changes Made

The files on irreducibles and polynomial functions have been corrected.

## Checklist

- [x] I have self-reviewed my code
- [x] Code follows project's style guidelines
- [x] Tests added and passing
- [x] Documentation updated (if needed)

## @Mention 

@Yag000

### CODE REVIEW

File: contents/anneaux_polynomes/irreductibilite.tex [hunk 1]
```diff
@@ -96,32 +96,32 @@ \subsection{Critères d'irréductibilité dans $K[X]$}
...
 	et donc $\bar{P} = \bar{Q}\bar{R}$ dans $\Zn{p}[X]$.
-	et de plus on a:
+	De plus, on a:
 	\begin{itemize}
 		\item $\deg Q = \deg \bar{Q}$
 		\item $\deg R = \deg \bar{R}$
 
 	\end{itemize}
-	Car leur coéffiients dominant ne divisen pas $p$.
+	car leur coefficients dominant ne divisent pas $p$.
```
Comment (Yag000) [inline contents/anneaux_polynomes/irreductibilite.tex]:
```suggestion
	car leurs coefficients dominants ne divisent pas $p$.
```

**Review by Yag000 — CHANGES_REQUESTED** [review]
Just a minor correction.

File: contents/anneaux_polynomes/irreductibilite.tex (hunk 1 above)
Comment (GabinDDL) [reply contents/anneaux_polynomes/irreductibilite.tex]: thanks, it's corrected

**Review by Yag000 — APPROVED** [review]
Thx :)

**Pull request merged**